.github/workflows/lint-and-test.yml

```diff
@@ -64,7 +64,9 @@ jobs:
         if: |
           contains(steps.list-changed.outputs.changed_charts, 'argocd-image-updater')
         run: |
-          kubectl apply -f charts/argo-cd/templates/crds
+          helm repo add dandydeveloper https://dandydeveloper.github.io/charts/
+          helm dependency build charts/argo-cd/
+          helm template charts/argo-cd/ -s templates/crds/* | kubectl apply -f -
```

Comment:
@bakito why so complicated now? ;-)

Reply:
AH lol, answering myself: Because Argo CD have now templated CRDs

Reply:
Because the crd need to be rendered since they are in templates.
Crds were moved into templates and extenedt with template logic which prevents them to be applied as is.

See https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/crds/crd-project.yaml#L6

To get my pr green I had to fix this logic in the action, as it was forgotten to be tested in the original PR.
https://github.com/argoproj/argo-helm/pull/1342

I did not find an easier solution, if you have one, feel free to change it.